### DISCUSSION TITLE
Possibility to add a "Country" column?

### DISCUSSION THREAD

**Discussion by arblad1990** [UnamSanctam/UnamWebPanel#197] [Ideas]
Hey love the web panel so far, you've done a great job on it. I was wondering if it was possible to get a column for "Country" as a dimension in the table? So that we could filter or sort by the estimated country of each IP? The reason for this is that I like to configure each individual IP to their local pool so that if I have a miner with an IP from Denmark I can configure it so that it uses the EU pool, or if I have one from India then I configure it to the Asia pool, at the moment I do this by manually looking up each IP so it would be great to have this as a feature :)

**Comment by UnamSanctam** [comment]
Yes that is planned (to show flags next to the IP), only problem is that the IP location databases are quite difficult to use without using composer from what I know so it was delayed.
